### ISSUE TITLE
Missing .mrmurano directory at setup

### ISSUE THREAD

**Issue by dweaver** [tadpol/MrMurano#2]
Following the setup instructions I get an error. Should I need to create the `.mrmurano` directory? 

```
$ gem install MrMurano
Fetching: commander-4.4.0.gem (100%)
Successfully installed commander-4.4.0
...
$ mr config solution.id 95m2GTqNqm
$ mr syncdown -V
Account name: danweaver@exosite.com
Password:  ***************
error: No such file or directory @ rb_sysopen - /Users/danw/.mrmurano/passwords. Use --trace to view backtrace
```

**Comment by tadpol** [comment]
Nuts.  Yes, it should have auto created the ~/.mrmurano directory.  You can create it by hand for now.

**Comment by tadpol** [comment]
Fix in 7b92387
